evm-ds/src/pretty_printer.rs

```diff
@@ -0,0 +1,88 @@
+use primitive_types::H256;
+use protobuf::Message;
+use log::debug;
+use std::fmt::Write;
+
+use crate::protos::{Evm as EvmProto, ScillaMessage};
+
+fn address_to_string(address: &EvmProto::Address) -> String {
```

Comment:
I think you could just use the `Into` trait implementations from `convert.rs`, and not duplicate code.